.travis.yml

```diff
@@ -46,11 +46,17 @@ install:
     fi
   - pwd
   - ls
+  # check if there is a partial download of pyglow, remove if true
+  - if [[ -d "$pyglow_dir" && ! -e "$pyglow_dir/setup.py" && "$TRAVIS_PYTHON_VERSION" == "2.7" ]]; then

```

Comment:
Is this check sufficient to ensure that all of pyglow has been downloaded?  Would it be safer to check directory size or something?